src/pkg/packager/prepare.go

```diff
@@ -28,9 +28,10 @@ import (
 )
 
 // FindImages iterates over a Zarf.yaml and attempts to parse any images.
-func (p *Packager) FindImages(baseDir, repoHelmChartPath string, kubeVersionOverride string) error {
+func (p *Packager) FindImages(baseDir, repoHelmChartPath string, kubeVersionOverride string) ([]string, error) {
```

Comment:
Instead of returning an `[]string` it would be more useful to return either a map of `component name --> []string`, or an `[]ZarfComponent` I believe this more properly fulfills the intent of the original issue.

Reply:
Agreed, makes sense

Reply:
Let me know if the change addresses your recommendation sufficiently.